userdatamodel/user.py

```diff
@@ -73,7 +64,7 @@ class User(Base):
     phone_number = Column(String)
     email = Column(String)
 
-    policies = relationship("Policy", secondary=users_to_policies)
+    _last_auth = Column(DateTime(timezone=False), server_default=func.now())
```

Comment:
I didn't find `_last_auth` in migrations?

Reply:
Actually really not sure what this is doing here. Will remove.

Reply:
Hah, Okie!